### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by ocharles** [fimad/prometheus-haskell#32]
Hi @fimad,

How are we looking on a new release? Anything I can do to help?

**Comment by fimad** [comment]
Do we want to include #26 and/or #29? If not, then I think there's nothing stopping us from cutting a new release.

**Comment by ocharles** [comment]
I think it'd be good to get those in, too. I'll get a fix up for #26 and finish off the work on #29. There's actually one more change to the WAI middleware that I need, which is to be able to configure the buckets used by the timing histograms.

**Comment by ocharles** [comment]
With such a collection of changes including the new `Metrics` API, I wonder if we bump the first version component and go to 1.0.

**Comment by fimad** [comment]
Sounds good to me.

**Comment by ocharles** [comment]
I'll try and get these fixed up this week. Would be really good to finish all this work!

**Comment by rkrzr** [comment]
I'd also be interested in a new release, esp. for the GHC 8.4 support.
I'm currently building directly from this repo using this `stack.yaml` file:

```yaml
resolver: lts-12.5

extra-deps:
  - github: fimad/prometheus-haskell
    commit: e44ab37a5d4e4a570e24cb1ca29d2753a27090c5
    subdirs:
     - prometheus-client
     - prometheus-metrics-ghc
     - wai-middleware-prometheus
```

**Comment by ocharles** [comment]
[I've released 1.0.0](https://hackage.haskell.org/package/prometheus-client-1.0.0)
